README.md

```diff
@@ -139,9 +140,47 @@ If you didn't use Raiden before, you can
 * Learn more by watching explanatory [videos](https://www.youtube.com/channel/UCoUP_hnjUddEvbxmtNCcApg)
 * Read the blog posts on [Medium](https://medium.com/@raiden_network)
 
-### Play with the Raiden Demo dApp
+### Try Out the Raiden Demo dApp
 
-You can play with the Raiden Demo dApp on any Ethereum Testnet hosted at https://lightclient.raiden.network/.
+These step-by-step instructions will guide you through the process for trying the Raiden dApp. We will be using the Ropsten testnet and MetaMask wallet in this example.
+
+__Prerequisites__
+
+You need to have MetaMask installed for your browser.
+
+1. Visit the [MetaMask website](https://metamask.io/) to download and install MetaMask.
+2. Select the Ropsten Test Network when you've logged in to your MetaMask.
+
+
+__Step 1: Acquire and Wrap ETH__
+
+1. Visit the Ropsten faucet at [https://faucet.ropsten.be](https://faucet.ropsten.be).
+2. Enter your keystore address associated with the Ropsten Test Network.
```

Comment:
I guess `wallet address` instead of `keystore address`

Reply:
Yes, I was thinking about wallet but was a bit unsure whether to use that term since I've only used "keystore address" previously. But I guess this confirms that wallet is a valid term. I will go ahead and change to wallet. :)